client/src/main/python/slipstream/executors/node/NodeDeploymentExecutor.py

```diff
@@ -144,8 +144,9 @@ def onSendingReports(self):
                              verboseThreshold=util.VERBOSE_LEVEL_QUIET)
 
     def _execute_report_target_and_send_reports(self):
+        exports = {'SLIPSTREAM_REPORT_DIR': util.get_platform_reports_dir()}
         try:
-            self._execute_target('report', ssdisplay=False, ignore_abort=True)
+            self._execute_target('report', exports=exports, ssdisplay=False, ignore_abort=True)
```

Comment:
Exactly.  That is why sometime ago I've added this 
https://github.com/slipstream/SlipStreamClient/blob/master/client/src/main/python/slipstream/util.py#L150